micrometer-observation/build.gradle

```
@@ -21,6 +25,7 @@ dependencies {
     optionalApi 'javax.servlet:javax.servlet-api'
 
     // Aspects
+    implementation "org.aspectj:aspectjrt:${libs.versions.aspectjweaver.get()}"
```

we can also replace this with a reference like in `micrometer-core`

```suggestion
    implementation libs.aspectjrt   
```